packages/updating-element/src/updating-element.ts

```diff
@@ -436,13 +449,52 @@ export abstract class UpdatingElement extends HTMLElement {
       ];
       // This for/of is ok because propKeys is an array
       for (const p of propKeys) {
-        // note, use of `any` is due to TypeSript lack of support for symbol in
+        // note, use of `any` is due to TypeScript lack of support for symbol in
         // index types
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         this.createProperty(p, (props as any)[p]);
       }
     }
     this.elementStyles = this.finalizeStyles(this.styles);
+    // DEV mode warnings
+    if (DEV_MODE) {
+      const warnRemoved = (obj: any, base: any, name: string) => {
+        if (obj[name] !== base[name]) {
```

Comment:
It was a little confusing that this isn't just a `obj[name] !== undefined` check, given that the warning is that "the API has been removed". 

This checks that there's not an _override_ on the user class, not that it doesn't exist at all. Is there a case where it's implemented on the base class (not removed altogether) and you're just not supposed to override it?

Reply:
Done